examples/03-Maxwell/Maxwell3D_Team3_bath_plate.py

```diff
@@ -152,8 +152,8 @@
 # Adjust eddy effects for the ladder plate and the search coil. The setting for
 # eddy effect is ignored for the stranded conductor type used in the search coil.
 
-m3d.eddy_effects_on(object_list=["LadderPlate"], activate_eddy_effects=True, activate_displacement_current=True)
-m3d.eddy_effects_on(object_list=["SearchCoil"], activate_eddy_effects=False, activate_displacement_current=True)
+m3d.eddy_effects_on(objects=["LadderPlate"], eddy_effects=True, displacement_current=True)
```

Comment:
enable_eddy_effects instead of eddy_effects
enable_displacement_current instead of displacement_current